getsolus/packages
# gnucash: Update to 5.4

**Summary**

- Full changelog can be found [here](https://raw.githubusercontent.com/Gnucash/gnucash/stable/NEWS)

**Test Plan**

- Open existing file(s) and make sure everthing ok
- Create new files

**Checklist**
